python/school-timetabling/static/index.html

```diff
@@ -2,7 +2,7 @@
 <head>
   <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
   <meta content="width=device-width, initial-scale=1" name="viewport">
-  <title>School timetabling - Timefold Quarkus</title>
+  <title>School timetabling - Timefold Python</title>
```

Comment:
```suggestion
  <title>School timetabling - Timefold Solver for Python</title>
```

Reply:
Thoughts on `Timefold Quarkus` in the Java quickstarts?

- `Timefold Solver in Quarkus`
- `Timefold Solver for Quarkus`

Reply:
Timefold Solver on Quarkus? With Quarkus? 
Either is better than "Timefold Quarkus", IMO.